static/include/components/board-attr-config.md

```diff
@@ -0,0 +1,60 @@
+Copy and paste the following attribute template into your board's **Attributes** box.
+Then remove and fill in the attributes as applicable to your board, according to the table below.
```

Comment:
```suggestion
Edit and fill in the attributes as applicable to your board, according to the table below.
```
(opt) Just to keep wording consistent

Reply:
I changed it for these ones with the non-fancy configs because if we're telling people to copy paste a template that has all attributes, even the optional and mutually-exclusive ones in it, I want to make it explicit that they can and should remove some of them.